theseus/labs/lie/functional/so3_impl.py

```diff
@@ -214,22 +207,23 @@ def _exp_impl(tangent_vector: torch.Tensor) -> torch.Tensor:
     one_minus_cosine_by_theta2 = torch.where(
         near_zero, 0.5 * sine_by_theta, (1 - cosine) / theta2_nz
     )
+
+    batch = tangent_vector.shape[:-1]
```

Comment:
Maybe `batch_shape` is a better name.

Reply:
Rename 'batch->size' as discussed offline.